harper-core/tests/text/linters/Part-of-speech tagging.snap.yml

```diff
@@ -165,6 +165,16 @@ Suggest:
 
 
 
+Lint:    Miscellaneous (10 priority)
+Message: |
+      65 | the earlier Brown Corpus and LOB Corpus tag sets, though much smaller. In
+      66 | Europe, tag sets from the Eagles Guidelines see wide use and include versions
+         |                           ^~~~~~ Use an apostrophe and `s` to form a noun’s possessive.
```

Comment:
This would seem to be a false positive beyond our control. `Eagles Guidelines` is some kind of named entity though so you could have a rule to not flag when the two words are capitalized in a sentence that is not in title case or all caps.